internal/operands/vm-console-proxy/reconcile_test.go

```diff
@@ -35,12 +40,20 @@ const (
 )
 
 var _ = Describe("VM Console Proxy Operand", func() {
-	var v *vmConsoleProxy
-	var mockedRequest *common.Request
+	const (
+		replicas int32 = 1
+	)
+
+	var (
+		bundle  *vmConsoleProxy
+		operand *vmConsoleProxy
+		request common.Request
+	)
 
 	BeforeEach(func() {
-		v = getMockedVmConsoleProxyOperand()
-		mockedRequest = getMockedRequest()
+		bundle = New(getMockedTestBundle())
+		operand = getMockedVmConsoleProxyOperand()
```

Comment:
Please use `New()` to create the operand. It doesn't make sense to mock and object that is being tested.